src/lib/stores/walletStore.js

```diff
@@ -95,19 +133,42 @@ function createWalletStore() {
          */
         sign: async ({ transactionXDR, network, pincode }) => {
             try {
-                const keyManager = setupKeyManager()
-                let signedTransaction = await keyManager.signTransaction({
+                const keyManager = setupKeyManager();
```

Comment:
We could move this line into the `else` block where we are signing with the pincode. Since it's not necessary when we are using the walletKit